lib/ansible/modules/network/dnac/dnac_rest.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Kevin Breit (@kbreit) <kevin.breit@kevinbreit.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: dnac_rest
+
+short_description: Direct access to the Cisco Software-Defined Access (SDA) controller, DNA Central
+
+version_added: "2.6"
+
+description:
+    - Enables management of the Cisco Software-Defined Access controller, more commonly called DNA Center.
+    - More information about the DNA Center API can be found at U(https://developer.cisco.com/site/dna-center-rest-api/).
+
+options:
+    username:
+        description:
+            - The username for use in HTTP basic authentication.
+        required: true
+    password:
+        description:
+            - The password for use in HTTP basic authentication.
+        required: true
+    host:
+        description:
+            - FQDN or IP address of DNA Central server.
+        required: true
+    method:
+        description:
+            - The HTTP method of the request.
+            - Using C(delete) is typically used for deleting objects.
+            - Using C(get) is typically used for querying objects.
+            - Using C(post) is typically used for modifying objects.
+        required: true
+        choices: [ delete, get, post ]
+    path:
+        description:
+            - Directory path to the endpoint. Do not include FQDN specified in C(host).
+        required: true
+    timeout:
+        description:
+            - HTTP timeout value.
+        default: 30
+    use_proxy:
+        description:
+            - If C(no), it will not use a proxy, even if one is defined in an environment variable on the target hosts.
+        type: bool
+        default: 'false'
```

Comment:
This needs to be:
```yaml
    default: 'no'
```
Because we want to documentation to clearly state yes/no. If you don't use quotes, YAML will consider it a boolean, and it will become True/False. (`type: bool` btw translates into `choices: [ 'no', 'yes' ]`)